frontend/constants/measurements.js

```diff
@@ -18,3 +18,9 @@ export const SEARCH_BAR_MOBILE_HEIGHT = '91.33px'
 
 export const ANIMATION_DURATION = '200ms'
 export const LONG_ANIMATION_DURATION = '400ms'
+
+export const M0 = '0'
```

Comment:
Maybe leave a TODO to standardize margins/padding in the codebase using these constants

Reply:
Clubhouse card should suffice. Used this type of syntax on some other projects recently and it's been very useful for standardizing things. Adding the card rn.

Reply:
https://app.clubhouse.io/pennlabs/story/1577/standardize-margins-across-the-site